src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
+    this.name = name;
+    this.health = health;
+    this.hidden = hidden;
+    Animal.alive.push(this);
```

Comment:
The constructor of the Animal class should not include the 'hidden' property as it is specific to Herbivores only. The 'hidden' property should be defined exclusively in the Herbivore class constructor.